action.yml

```diff
@@ -24,5 +28,30 @@ inputs:
     default: 'gpt-4-turbo-preview'
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile' 
\ No newline at end of file
+  using: 'composite'
+  steps:
+    - name: Set up Python
+      uses: actions/setup-python@v5
```

Comment:
📦 The 'Set up Python' step has been added, which is essential for the action's execution. Ensure the specified Python version is compatible with the action's requirements.